modules/veteran_verification/VETERAN_VERIFICATION.yml

```diff
@@ -88,6 +88,8 @@ paths:
                 properties:
                   data:
                     $ref: '#/components/schemas/VeteranStatusConfirmation'
+        '401':
+          description: Unauthorized.
         '403':
```

Comment:
I just tested out this scenario and it actually is returning a 401 Not Authorized as well if the required scope isn't provided.

Recommend removing the 403 block.